File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains
with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("Title is: "+driver.title);
    actions=ActionChains(driver)
    # Perform a LEFT CLICK action
    actions.click().perform()
    active_side=driver.find_element(By.CLASS_NAME,"active")
    print("active side number after left click :"+active_side.text)

    # perform Double click action
    actions.double_click().perform()
    active_side=driver.find_element(By.CLASS_NAME,"active")
    print("active side number after double click :"+active_side.text)

    # Perform a RIGHT CLICK action
    actions.context_click().perform()
    active_side=driver.find_element(By.CLASS_NAME,"active")
    print("active side number after Right click :"+active_side.text)

File: Selenium/Activities/Activity5.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity5 {
    public static void main(String[] args) {
        WebDriver driver=new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/input-events");
        System.out.println("Title is :"+driver.getTitle());
        Actions actions=new Actions(driver);
        // Perform left click
        actions.click().pause(1000). build().perform();
        String frontText=driver.findElement(By.className("active")).getText();
        System.out.println("front text is :"+frontText);

        //Perform double click
        actions.doubleClick().pause(1000).build().perform();
        frontText=driver.findElement(By.className("active")).getText();
        System.out.println("front text is :"+frontText);

        // Perform right click
        actions.contextClick().pause(1000).build().perform();
        frontText=driver.findElement(By.className("active")).getText();
        System.out.println("front text is :"+frontText);
        driver.close();
    }
}
;
